loadtesting/proxy/tests/scheduled_tx_uniswap.py

```diff
@@ -0,0 +1,361 @@
+        program_id=network_object["evm_loader"],
+        endpoint=network_object["solana_url"],
+        neon_chain_id=network_object["network_ids"]["neon"],
+        sol_chain_id=network_object["network_ids"]["sol"],
+        neon_token_mint_str=network_object["spl_neon_mint"],
+    )
+
+    # create solana account
+    solana_account = Keypair()
+    fund_solana_account(environment.evm_loader, solana_account, bank_account, network, network_object)
+    environment.solana_account = solana_account
+
+    # create neon accounts
+    neon_accounts = []
+    for i in range(neon_users):
+        neon_solana_account = Keypair()
+        LOG.info(f"Creating {i} neon user for uniswap...")
+        neon_user = NeonUser(environment.evm_loader.loader_id, keypair=neon_solana_account)
+        neon_accounts.append(neon_user)
+        fund_solana_account(environment.evm_loader, neon_user.solana_account, bank_account, network, network_object)
+
+    test_tokens = ["TTA", "TTB", "WETH"]
+    tokens = {}
+
+    # deploy erc20 tokens
+    for token in test_tokens:
+        LOG.info(f"Start to deploy token {token}...")
+        eth_account = account_manager.create_account()
+        erc20 = ERC20Wrapper(
+            web3_client,
+            faucet,
+            f"Test {token}",
+            token,
+            environment.evm_loader,
+            solana_account=solana_account,
+            mintable=True,
+            bank_account=bank_account,
+            account=eth_account,
+        )
+        tokens[f"{token}"] = erc20
+
+        LOG.info("Mint tokens...")
+        erc20.mint_tokens(signer=erc20.account, to_address=erc20.account.address, amount=token_mint_amount)
+
+        resp = erc20.mint_tokens(signer=erc20.account, to_address=deployer.address, amount=initial_amount)
+        assert resp["status"] == 1
+
+        LOG.info("Mint tokens to neon accounts for swaps...")
+        receipts = []
+        for neon_user in neon_accounts:
+            receipts.append(
+                erc20.mint_tokens(signer=erc20.account, to_address=neon_user.checksum_address, amount=token_mint_amount)
+            )
+
+        for item in receipts:
+            assert item["status"] == 1
+
+    LOG.info("Deploy UniswapV3Factory...")
+    uniswap_v3_factory, _ = web3_client.deploy_and_get_contract(
+        "external/uniswap-v3/contracts/UniswapV3Factory", "0.7.6", account=deployer
+    )
+
+    LOG.info("Deploy SwapRouter...")
+    swap_router, _ = web3_client.deploy_and_get_contract(
+        "external/uniswap-v3/contracts/SwapRouter",
+        "0.7.6",
+        account=deployer,
+        constructor_args=[uniswap_v3_factory.address, tokens["WETH"].contract_address],
+        import_remapping=REMAPPING_ZEPPELIN_UNISWAP,
+    )
+
+    LOG.info("Approve for swap router...")
+    receipts = []
+    receipts.append(tokens["TTA"].approve(deployer, swap_router.address, token_mint_amount))
+    receipts.append(tokens["TTB"].approve(deployer, swap_router.address, token_mint_amount))
```

Comment:
I think you don't need these 2 approves as you do it inside scheduled trx by neon users